### ISSUE TITLE
Issue on length of merged INS 

### ISSUE THREAD

**Issue by xinbeibei** [fritzsedlazeck/SURVIVOR#108]
Hi Fritz,

We have multiple samples, all samples has an insertion starting from the same position, and most samples has SVLEN=600bp (PBSV result), however, only one sample has the SVLEN=7000bp. After merging using SURVIVOR with the following command: 
`SURVIVOR merge INS_for_SURVIVOR.lst 1000 1 1 -1 -1 -1 INS.samples.vcf`
the length for merged SV is 7000bp. Since there is only one sample supporting 7000bp, is there any parameter to constrain the number of samples supporting a SV? Or report two SVs at this position, one with 600bp and the other one wiht 7000bp? 

Thanks and looking forward,
Beibei

**Comment by fritzsedlazeck** [comment]
Hi Beibei,
the other insertions are reported in the genotype field. I would encourage you to download the newest source from github and try this again.  The two insertions (7000bp . vs. 600) should not be merged anymore. 
Thanks
Fritz

**Comment by xinbeibei** [comment]
Thanks, Fritz, I will download the latest version and merge again.

**Comment by xinbeibei** [comment]
Hi Fritz,

As you suggested, we tried different pairwise distance today with the latest SURVIVOR. When using default pairsewise distance (PD=1kb), we got only one 7kbp INS. When using PD=500bp, we got two INSs at the same location, 600bp & 7Kbp. Is there any way to only get 600bp as merged SV for this location, since 7kbp is only supported by one sample? If not and if we still want to get 600bp INS, what PD will you suggest us to use?

Thank you very much,
Beibei

**Comment by fritzsedlazeck** [comment]
Mhm the problem is that survivor ranks all candidates based on the start location and pick the one with the median start location as a good representative. Apparently that is your larger ins...

Can you share with me these calls so I can see what I can do? Fritz.sedlazeck@gmail.com

Thanks
Fritz

**Comment by xinbeibei** [comment]
Hi Fritz,

I just sent you the test file that we used. Look forward to your solution.

Thanks again,
Beibei

**Comment by fritzsedlazeck** [comment]
Hi Beibei,
happy new year and thanks for these. I got them. Give me some time I just returned from the holidays.
Thanks
Fritz

**Comment by MaestSi** [comment]
Hi Fritz,
I am commenting here as I think the issue I am encountering is related to this one.
I am merging SVs from 3 platforms, ignoring SV type and considering 50kbp as the breakpoint tolerance.
In particular, one of my entries in the merged vcf is the following:
`chr02   10940395        SMAP228 N       <INS>   180     PASS    SUPP=3;SUPP_VEC=111;SVLEN=-26310;SVTYPE=INS;SVMETHOD=SURVIVOR1.0.7;CHR2=chr02;END=10959195;CIPOS=0,24282;CIEND=0,38843;STRANDS=+- GT:PSV:LN:DR:ST:QV:TY:ID:RAL:AAL:CO     0|1:NA:37424:0,0:+-:180:DEL:call_133:NA:NA:chr02_10964677-chr02_11002101        1/1:NA:4071:0,0:+-:20:INS:SMAP228:NA:NA:chr02_10940395-chr02_10959195     0/1:NA:37435:33,26:+-:.:DEL:6719:NA:NA:chr02_10964674-chr02_11002109`

As `chr02_10964674-chr02_11002109` is the SV with the median start location, I would have expected this one to be picked as the representative, instead of `chr02_10940395-chr02_10959195`. What do you think?
Thanks,
Simone

**Comment by fritzsedlazeck** [comment]
Yeah I need to dig into this. I am sorry that it takes so long, just two grant submissions this week.. so should have time for this very soon.
Sorry and happy new year. 
Fritz

**Comment by zhishanchen** [comment]
Hi Fritz,

Happy new year!
A similar issue also happened for me. I was merging SVs from six tools. I only considered the SV with length of > 50bp, so I used the following parameters: 1000 2 1 1 0 50.
After merging, I noticed I still got some SVs with length of < 50 bp, like the example below:
  
chr1    111543603       DEL00310SUR     GAGGAGGAGGAAGAGAAGGAGGAAT       G PASS    SUPP=2;SUPP_VEC=010010;SVLEN=-38;SVTYPE=DEL;SVMETHOD=SURVIVOR1.0.6;CHR2=chr1;END=111543627;CIPOS=0,0;CIEND=0,1;STRANDS=+-       GT:PSV:LN:DR:ST:QV:TY:ID:RAL:AAL:CO     ./.:NaN:0:0,0:--:NaN:NaN:NaN:NAN:NAN:NAN       1/1:NA:39:0,31:+-:.:DEL:DEL00000624:GAGGAGGAGGAAGAGAAGGAGGAAT:G:chr1_111543603-chr1_111543628   ./.:NaN:0:0,0:--:NaN:NaN:NaN:NAN:NAN:NAN        ./.:NaN:0:0,0:--:NaN:NaN:NaN:NAN:NAN:NAN        0/1:NA:37:0,0:+-:.:DEL::GAGGAGGAGGAAGAGAAGGAGGAAT:G:chr1_111543603-chr1_111543627       ./.:NaN:0:0,0:--:NaN:NaN:NaN:NAN:NAN:NAN


Do you have any idea for this issue?

Thank you so much!

Zhishan
